Dandelion Sprout's Anti-Malware List.txt

```diff
@@ -1682,6 +1682,24 @@ zombooru.com##a[href="http://www.hard55.com"]
 ||dl02.s3.amazonaws.com/installers/747947/oi_picasa38-setupexe.exe^$all
 ! https://www.virustotal.com/gui/file/2ea599605c4d65902943f12e1114a71af7a40fa7dffbf018b0ee3e7a61aaeaa3/community
 ||dl02.s3.amazonaws.com/installers/424531/2gzbsoj4gxb.exe^$all
+! https://www.huorong.cn/info/1531309921141.html
+||kuaizip.com^$all
+! https://www.huorong.cn/info/1618397948649.html
+! ||zhuangjizhuli.com^$all
```

Comment:
Why is this commented out?

Reply:
I think there is still a possibility that this domain will become a malicious domain (to avoid detection by security vendors).